### DISCUSSION TITLE
Filter vertically strings

### DISCUSSION THREAD

**Discussion by ghost** [jsvine/pdfplumber#371] [Ask for help with specific PDFs]
Hello dear developers,
is there a possibility to ignore a string that is printed vertically on each page in the lower left corner? Currently the single letters/digits are split and set in front of the corresponding lines.
Thank you in advance for your guidance.

**Comment by jsvine** [comment]
It's fairly difficult to provide a confident answer without seeing the specific PDF in question, but you might try using `page.crop` to focus just on the core area of the PDF (see documentation in README.md), or `page.filter(lambda x: x.get('upright') != 0)` to remove vertical characters.
